test/Microsoft.AspNetCore.Mvc.ViewFeatures.Test/Internal/PartialViewResultTest.cs

```diff
@@ -23,6 +23,23 @@ namespace Microsoft.AspNetCore.Mvc.ViewFeatures.Internal
     public class PartialViewResultTest
```

Comment:
Don't currently have a `ViewResultTest`. Please file a follow-up issue to create one w/ similar tests to those in this class.